docker/sparkswapd/Dockerfile

```diff
@@ -39,6 +42,15 @@ COPY --from=builder /home/app/proto /home/app/proto
 
 # Copy node_modules
 COPY --from=builder /home/app/node_modules /home/app/node_modules
+COPY --from=builder /home/app/broker-cli/node_modules /home/app/broker-cli/node_modules
+
+ARG USER_HOME=/root
```

Comment:
add a comment about why we need this, e.g. "Broker CLI expects config to be at ~/.sparkswap, so we need the default docker user's home directory to install our container config into"